jsettlers.graphics/src/jsettlers/graphics/map/controls/original/panel/content/Label.java

```diff
@@ -20,12 +20,18 @@
 import jsettlers.graphics.utils.UIPanel;
 
 public class Label extends UIPanel {
+    public enum HorizontalAlignment {
+        LEFT,
+        CENTRE,
+        RIGHT,
+    }
 
 	private final EFontSize size;
-	private final String[] words;
+	private String[] words;
```

Comment:
In case you missed it, I did add the change to avoid any concurrency issues in the last update so it should be good to go.